### ISSUE TITLE
Joomla 4 can't login to admin

### ISSUE THREAD

**Issue by stankurd** [joomla/joomla-cms#21334]
### Steps to reproduce the issue
https://screenshots.firefox.com/6oN3bXeROeXimVrr/127.0.0.1

**Comment by ghost** [comment]
Please give "System information (as much as possible)"

**Comment by stankurd** [comment]
i have this bug after update to last git revision

Setting | Value
-- | --
 
Windows NT OLA-TOSH 6.1 build 7601 (Windows 7 Home Premium Edition Service Pack 1) AMD64
mysql
5.7.9
utf8_general_ci
utf8_general_ci
7.1.9
Apache/2.4.17 (Win64) PHP/7.1.9
apache2handler
Joomla! 4.0.0-alpha5-dev Development [ Amani ] 22-July-2018 22:58 GMT
Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.75 Safari/537.36

**Comment by jeckodevelopment** [comment]
did you use the nightly version?

**Comment by ghost** [comment]
@jeckodevelopment update Today on Nightly without issue.
### System information
- Nightly Build 4.0.0-alpha5-dev
- Multilingual Sample Data (French, German DE, Persian)
- Template: Cassiopeia
- macOS Sierra, 10.13.6
- Firefox 61 (64-bit)
#### CloudAccess.net
- PHP 7.1.15
- MySQLi 5.7.18-cll-lve

**Comment by stankurd** [comment]
I have update Joomla from git with last changes

**Comment by brianteeman** [comment]
Because of recent changes the git branch is NOT installable without extra steps.

**Comment by brianteeman** [comment]
As the updated instructions have not been merged yet here they are

https://github.com/dgrammatiko/joomla-cms/blob/6a7790a4aacca2013219f640211a0aa1bf987fd6/README.md

**Comment by ElephantInTheRoomFirst** [comment]
Today I have installed latest J4 build. After installation I was able to login. During the session I had enabled double authentication and disabled them. But in database the dual authentication was active. So I deactivated them in database.
Then I tried to login in admin panel, unfortunately it did not work. So modified the password in database using MD5 encrypt md5online.org. I was not able to login and the MD5 value in database was reset to random value. This happed couple to times before also with double authetication scenario. The same thing J3 works fine.
